javascript/extractor/lib/typescript/src/type_table.ts

```diff
@@ -766,6 +766,15 @@ export class TypeTable {
     return hash;
   }
 
+  /** Returns the type of `symbol` or `null` if it could not be computed. */
+  private tryGetTypeOfSymbol(symbol: ts.Symbol) {
+    try {
+      return this.typeChecker.getTypeOfSymbolAtLocation(symbol, this.arbitraryAstNode)
+    } catch (e) {
+      return null;
```

Comment:
No warning printed in this case?

Reply:
Added a warning, though it's not as useful since we have no location information here. For one of the bugs you linked to, it just prints `Could not compute type of 'widget'`.

Reply:
Great; (very slightly) better than nothing, I think.